reportportal_listener/model.py

```diff
@@ -97,11 +97,17 @@ def get_name(self):
         """Get keyword name.
 
         Returns:
-            Наименование, обрезанное до 256 символов.
+            Name, cropped up to 256 characters.
         """
-        assignment = u"{0} = ".format(", ".join(self.assign)) if self.assign else ""
-        arguments = u", ".join(self.args)
-        full_name = u"{0}{1} ({2})".format(assignment, self.name, arguments)
+        assign = ", ".join(self.assign).encode("utf8")
+        assignment = "{0} = ".format(assign) if self.assign else ""
+        arguments = ", ".join(self.args)
+        # full_name = "{0}{1} ({2})".format(assignment, self.name, arguments)
```

Comment:
Please remove this comment, there is no need for it.

Reply:
Hello, @ailjushkin ， thanks for your review. I have been updating my code to remove this comment “# full_name = "{0}{1} ({2})".format(assignment, self.name, arguments)”.